locale/ru/LC_MESSAGES/buku.po

```diff
@@ -0,0 +1,873 @@
+"                         аргумент отсутствует.\n"
+"                         в противном случае пишет в данный файл\n"
+"    --colors COLORS      установить выходные цвета в пятибуквенной\n"
+"                         строке\n"
+"    --nc                 отключить цветной вывод\n"
+"    -n, --count N        показывать N результатов на странице\n"
+"                         (по умолчанию 10)\n"
+"    --np                 не показывать подсказку, запустить и выйти\n"
+"    -o, --open [...]     просматривать закладки по индексам и\n"
+"                         диапазонам\n"
+"                         открыть случайную закладку, если нет\n"
+"                         аргументов\n"
+"    --oa                 просмотреть все результаты поиска сразу\n"
+"    --replace старый новый\n"
+"                         заменить везде старый тег новым тегом\n"
+"                         удалить старый тег, если новый тег не указан\n"
+"    --shorten индекс|URL получить укороченный URL из сервиса tny.im\n"
+"    --expand индекс|URL  расширить сокращенный URL tny.im\n"
+"    --cached индекс|URL  просмотреть кешированную страницу из\n"
+"                         Wayback Machine\n"
+"    --suggest            показывать похожие теги при добавлении\n"
+"                         закладок\n"
+"    --tacit              уменьшить многословие, пропустить некоторые\n"
+"                         подтверждения\n"
+"    --nostdin            не ждать ввода (должен быть первым\n"
+"                         аргументом)\n"
+"    --threads N          максимальное количество сетевых соединений\n"
+"                         при полном обновлении\n"
+"                         по умолчанию N=4, мин N=1, макс N=10\n"
+"    -V                   проверьте последнюю доступную версию\n"
+"                         основной ветки\n"
+"    -g, --debug          показать отладочную информацию и подробные\n"
+"                         журналы"
+
+#: buku:5591
+msgid "[7mbuku (? for help)[0m "
+msgstr "[7mbuku (? вызов справки)[0m "
+
+#: buku:5684
+msgid "Bookmark a single URL at a time"
+msgstr ""
+
+#: buku:5743 buku:5767
+msgid "no keyword"
+msgstr ""
+
+#: buku:5791
+msgid "no expression"
+msgstr ""
+
+#: buku:5828
+msgid "No search criteria to exclude results from"
+msgstr ""
+
+#: buku:5907
+msgid "Updated results:\n"
+msgstr ""
+
+#: buku:5970
+#, fuzzy
+msgid "Invalid index or range to update"
+msgstr "Недопустимый индекс или диапазон или комбинация"
+
+#: buku:5988
+#, fuzzy
+msgid "Invalid index or range to delete"
+msgstr "Недопустимый индекс или диапазон или комбинация"
+
+#: buku:6003
+msgid "Invalid index or range or combination"
+msgstr "Недопустимый индекс или диапазон или комбинация"
+
+#: buku:6028
+#, fuzzy
+msgid "Invalid index or range to print"
+msgstr "Недопустимый индекс или диапазон или комбинация"
+
+#: buku:6063
+#, fuzzy
+msgid "Invalid index or range to open"
+msgstr "Недопустимый индекс или диапазон или комбинация"
```

Comment:
The English versions for these five entries are different but the Russian is identical. Is it necessary to discard the differing details?